iis-workflows/iis-workflows-transformers/src/test/resources/eu/dnetlib/iis/workflows/transformers/spark/citationmatching/direct/sampledataproducer/oozie_app/workflow.xml

```diff
@@ -0,0 +1,100 @@
+<workflow-app xmlns="uri:oozie:workflow:0.5" name="test-transformers_citationmatching_direct (spark)">
```

Comment:
I'm not sure if the 'sampledataproducer' name makes sense here. What you test here is not some 'producer' but input transformer.